docker/openqa_data/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM opensuse:13.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com> Josef Skladanka <jskladan@redhat.com>
+
+RUN zypper --non-interactive --gpg-auto-import-keys in ca-certificates git wget
```

Comment:
curl https://build.opensuse.org/source/devel:openQA/_pubkey

You can use rpm --import on that